### PR TITLE
Beta doc 07

### PR DESCRIPTION
**Pull request by snehalkotwal**

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaFeatureRolloutPolicy.md [hunk 1]
```diff
@@ -111,11 +150,12 @@ Accept wildcard characters: False
 ```
 
 ### -Filter
-The oData v3.0 filter statement. 
+
+The oData v3.0 filter statement.
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaFeatureRolloutPolicy.md:154]:
```suggestion
The OData v4.0 filter statement.
```

**Review by KenitoInc — APPROVED** [review]

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaFeatureRolloutPolicyDirectoryObject.md [hunk 2]
```diff
@@ -8,36 +19,43 @@ schema: 2.0.0
...
+with Seamless single sign-on or not, or whether Passthrough auth or not). Specify `Id` and `RefObjectId` parameter to add a group to the cloud authentication roll-out policy.
 
 ## Examples
 
-### Example 1: Adds a group to the cloud authentication roll-out policy in Azure AD.
-```
-PS C:\> Add-EntraBetaFeatureRolloutPolicyDirectoryObject -Id "a03b6d9e-6654-46e6-8d0a-8ed83c675ca9" -RefObjectId "0a1068c0-dbb6-4537-9db3-b48f3e31dd76"
+### Example 1: Adds a group to the cloud authentication roll-out policy in Microsoft Entra ID
+
+```powershell
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaFeatureRolloutPolicyDirectoryObject.md:45]: Add the required scopes.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaFeatureRolloutPolicyDirectoryObject.md [hunk 3]
```diff
@@ -8,36 +19,43 @@ schema: 2.0.0
...
+Add-EntraBetaFeatureRolloutPolicyDirectoryObject -Id '00aa00aa-bb11-cc22-dd33-44ee44ee44ee' -RefObjectId 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaFeatureRolloutPolicyDirectoryObject.md:46]:
```suggestion
$params = @{
    Id = '00aa00aa-bb11-cc22-dd33-44ee44ee44ee'
    RefObjectId = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
}

Add-EntraBetaFeatureRolloutPolicyDirectoryObject @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaFeatureRolloutPolicy.md [hunk 4]
```diff
@@ -8,34 +19,40 @@ schema: 2.0.0
...
 
 ## Examples
 
-### Example 1: Removes the policy for cloud authentication roll-out in Azure AD.
-```
-PS C:\> Remove-EntraBetaFeatureRolloutPolicy -Id "7b10cf40-bc0e-46b5-9456-4520179eef5d"
+### Example 1: Removes the policy for cloud authentication roll-out in Microsoft Entra ID
+
+```powershell
+Remove-EntraBetaFeatureRolloutPolicy -Id '11bb11bb-cc22-dd33-ee44-55ff55ff55ff'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaFeatureRolloutPolicy.md:43]:
```suggestion
Connect-Entra -Scopes 'Directory.ReadWrite.All'
Remove-EntraBetaFeatureRolloutPolicy -Id '11bb11bb-cc22-dd33-ee44-55ff55ff55ff'
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaFeatureRolloutPolicyDirectoryObject.md [hunk 5]
```diff
@@ -8,36 +19,43 @@ schema: 2.0.0
...
 
 ## Examples
 
-### Example 1: Removes a group from the cloud authentication roll-out policy from Azure AD.
-```
-PS C:\> Remove-EntraBetaFeatureRolloutPolicyDirectoryObject -Id "a03b6d9e-6654-46e6-8d0a-8ed83c675ca9" -ObjectId "0a1068c0-dbb6-4537-9db3-b48f3e31dd76"
+### Example 1: Removes a group from the cloud authentication roll-out policy from Microsoft Entra ID
+
+```powershell
+Remove-EntraBetaFeatureRolloutPolicyDirectoryObject -Id '00aa00aa-bb11-cc22-dd33-44ee44ee44ee' -ObjectId 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaFeatureRolloutPolicyDirectoryObject.md:46]: Add required scopes.

Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaFeatureRolloutPolicyDirectoryObject.md:46]:
```suggestion
$params = @{
    Id = '00aa00aa-bb11-cc22-dd33-44ee44ee44ee'
    ObjectId = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
}

Remove-EntraBetaFeatureRolloutPolicyDirectoryObject @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaTrustFrameworkPolicy.md [hunk 6]
```diff
@@ -8,34 +19,39 @@ schema: 2.0.0
...
 
 ## Examples
 
-### Example 1
-```
-PS C:\> Remove-EntraBetaTrustFrameworkPolicy -Id B2C_1A_signup_signin
+### Example 1: Removes the specified trust framework policy
+
+```powershell
+Remove-EntraBetaTrustFrameworkPolicy -Id B2C_1A_signup_signin
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaTrustFrameworkPolicy.md:42]: Add required scopes.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaTrustFrameworkPolicy.md [hunk 7]
```diff
@@ -47,14 +63,17 @@ Accept wildcard characters: False
...
 
 ### System.String
+
 ## Outputs
 
 ### System.Object
+
 ## Notes
 
 ## Related Links
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaTrustFrameworkPolicy.md:79]: Add related cmdlets e.g. Update, Get, Add cmdlets.

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Review by KenitoInc — APPROVED** [review]

**Review by v-varshamane — APPROVED** [review]

**Pull request merged**